tests/GlobTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WP_Rocket\Tests;
+
+trait GlobTrait {
+
+	public function deleteDomainCallback( $root ) {
+		$root = rtrim( $root, '*' );
+		$this->delete_files( $root );
```

Comment:
Shouldn't it be `deleteFiles()`?
```suggestion
		$this->deleteFiles( $root );
```

Reply:
Good catch @Screenfeed. Committed your suggestion https://github.com/wp-media/wp-rocket/pull/2470/commits/c5c8f18c1da98d08b1617ce23032dbaa2ffe87ec